### ISSUE TITLE
Claim & Step Table Accessibility issues (multiple)

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#18160]
## What happened?
1. Android w/TalkBack enabled:  Touching anywhere on the the Step Table causes android to speak "Claims status screen."  It never reads the text for the individual steps, including dates like iOS does.

2. Android w/TalkBack enabled:  Mobile Alert announcing number of file requests from VA is not read.  It seems to be included in the "Claims status screen" announcement from #1 above.

3. Android/iOS w/Appropriate Screen reader enabled:  View File Requests button - Android read as "the button, claim phase, file request, button, a11y hint, button. Double tap to activate."  iOS read as "View File requests, button, claim phase, file requests, button, a11y hint."
Not entirely sure what it should say but this really isn't it.  Check with Bess?

4. Expand/collapse arrows need to have a wider tap target area. Minimum is 44px and currently it is 16px.

5. Icon "3" in this case, for step three, is read as "three" on iOS (I can't tell for Android because of #1 above).  When I tap on the check mark icon for completed previous steps, nothing is read.  There should always be a hint or something for this icon explaining what it is.  Here is #1 from the Accessibility Acceptance Criteria:  "Everything on the page should be read by the screen reader.
If anything is being conveyed visually only (buttons, menu items, list, tables, images, headings, forms, etc) the component type needs to be announced by the screenreader. (Include name (title), role, value)"

## Specs:
- Device: Android / iOS phone with appropriate screen reader software enabled.

## Steps to Reproduce
See above in "What happened?"

## Desired behavior
See above in "What happened?"

## Acceptance Criteria
See above in "What happened?"

## Bug Severity
- 4 - Low

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17734

## Screen shot(s) and additional information
![image.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/e84f2a44-e023-4fd9-8dd6-0280f03b2099)

**Comment by digitalken** [comment]
1/8/2021 - Closing.
